region/maridia/inner-pink/Aqueduct.json

```diff
@@ -1275,35 +1275,27 @@
       ]
     },
     {
-      "id": 52,
       "link": [2, 9],
       "name": "Suitless Shinespark",
       "entranceCondition": {
-        "comeInRunning": {
-          "speedBooster": true,
-          "minTiles": 2.4375
+        "comeInShinecharging": {
+          "length": 3,
+          "openEnd": 1
         }
       },
       "requires": [
         "canWaterShineCharge",
-        "canShinechargeMovement",
-        {"canShineCharge": {
-          "usedTiles": 25,
-          "openEnd": 2
-        }},
         {"shinespark": {"frames": 37, "excessFrames": 3}}
       ],
       "note": [
         "Enter this room while holding dash in the previous room, before the door transition.",
```

Comment:
```suggestion
        "Enter this room while holding dash in the previous room, before the door transition, using a 1-tap or a stutter.",
```
I think the `canShineCharge` and `canShinechargeMovement` requirements should probably be here? Assuming we're not doing a stutter, doing it without shinecharge movement (aside from turnaround) requires a fairly precise tap and down press which seems more difficult than doing a bit of movement?

Reply:
`comeInShinecharging` is used here with the intent that most of the shinecharge is completed before reaching the door.

Reply:
Ah, right. I was misreading it as though it were comeInRunning